modules/custom/engineering_profile_helper/modules/engineering_magazine/lib/scss/components/magazine_navigation/_magazine-navigation.scss

```diff
@@ -1,134 +1,143 @@
+      line-height: 1.55em;
+      color: #333333;
 
+      margin-right: 1em;
+      padding-bottom: 1em;
 
-        .magazine-landing-nav__item {
-            font-size: 2em;
-            font-weight: 400;
-            line-height: 1.55em;
-            color: #333333;
+      #magazine-landing-nav__topics-toggle {
+        display:inline-block;
+        position: relative;
+        width: 30px;
+        height: 30px;
+        img {
 
-            margin-right: 1em;
-            padding-bottom: 1em;
-        }
-    }
+          display:inline;
 
-    .magazine-landing-nav__topics_panel {
-        display: none;
-        clear:both;
-        margin-left: calc(50% - 50vw);
-        margin-right: calc(50% - 50vw);
-        width: 100%;
-        width: 100vw;
-        background-color: #575757;
-        padding: 2em;
-
-        .magazine-terms-list {
-            display: flex;
-            flex-wrap: wrap;
-            justify-content: space-around;
-
-            width: 75%;
-            margin-left:auto;
-            margin-right:auto;
-
-            .magazine-term {
-                flex: 1 1 25%;
-                display: inline-block;
-                text-align: center;
-                margin-top: .75em;
-                margin-bottom: .75em;
-                font-size: 1em;
-                font-weight: 400;
-                color: white;
-                text-decoration: none;
-
-                &:hover {
-                    text-decoration: underline;
-                }
-
-            }
         }
+      }
     }
-}
+  }
 
-._9b76306-c0b8-4dcd-8dde-562796493bab {
-    h2 {
-        font-size: 2.8em;
-        font-weight: 600;
-        letter-spacing: 0;
-        margin: -9px 0 10px;
-        text-align: center;
-        padding-top: 2em;
-    }
+  .magazine-landing-nav__topics-panel {
+    display: none;
+    clear: both;
+    margin-left: calc(50% - 50vw);
+    margin-right: calc(50% - 50vw);
+    width: 100vw;
+    background-color: #575757;
+    padding: 2em;
+
+    .magazine-terms-list {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-around;
 
-    h3 {
-        width: 40%;
-        color: #606060;
-        font-weight: 400;
-        letter-spacing: 0;
-        margin: 0 auto 60px;
+      width: 75%;
+      margin-left: auto;
+      margin-right: auto;
+
+      .magazine-term {
+        flex: 1 1 25%;
+        display: inline-block;
         text-align: center;
-    }
-}
+        margin-top: 0.75em;
+        margin-bottom: 0.75em;
 
-.magazine-topics__term-name {
-    width: 100%;
-    text-align: center;
+        a {
+          font-size: 1em;
+          font-weight: 400;
+          color: #ffffff;
+          text-decoration: none;
+
+          &:hover {
+            text-decoration: underline;
+          }
+        }
 
-    h2 {
-        font-size: 2.8em;
-        font-weight: 600;
-        letter-spacing: 0;
-        margin: -9px 0 10px;
+      }
     }
+  }
 }
 
+._9b76306-c0b8-4dcd-8dde-562796493bab {
```

Comment:
This is an unexpected selector. Is it generated? Will it remain as expected? Is there a comparable human readable one?

Reply:
turns out it wasn't needed.  it was from when I was prototyping.  fixed by excision.